neqo-transport/src/recovery/mod.rs

```diff
@@ -735,9 +735,14 @@ impl LossRecovery {
 
     /// Calculate when the next timeout is likely to be.  This is the earlier of the loss timer
     /// and the PTO timer; either or both might be disabled, so this can return `None`.
```

Comment:
This probably needs to be updated.

Reply:
What should it say instead? The
> either or both might be disabled, so this can return `None`

bit seems to cover what happens, even with the change?

Reply:
Yeah, you are right.  Nevermind.